docs/UserGuide.md

```diff
@@ -164,25 +170,35 @@ To view all our features, you may visit out [features section](#features).
 
 ### Adding an assignment
 
-```put command first```
+```add-a n/NAME [d/DESCRIPTION] [s/STATUS] e/YYYY-MM-DD HH:mm [p/YYYY-MM-DD HH:mm]```
 
 **Purpose:**
-
+To add an assignment into CampusCompanion.
 <div markdown="block" class="alert alert-danger">
 
 **ALERT**
-Put alerts here, if any
+This command cannot be undone. If you add in the wrong details, 
+you would have to delete the assignment using `delete-a` and add it again. If instead
+your status or description is wrong, you can simply `mark-a`/`unmark-a` or `edit-a`.
+HH:mm is set to 23:59 by default if left empty.
 </div>
 
 <div markdown="block" class="alert alert-info">
 
 **Note**
-Put notes here
+- The name(n/) and deadline(e/) of the assignment are compulsory, the 
+description(d/), status(s/) and planned finish date(p/) are all optional fields.
+- The deadline(e/) and planned end date(p/) **must** be after the current date and time.
+Otherwise, an error message will be shown.
 </div>
 
-**Examples:** show some examples with the parameters
+**Examples:** 
+- `add-a n/CS2100 Assignment 1 e/2023-10-29 13:00`
+- `add-a n/CS2103T v1.3 d/Work on add function for task e/2023-10-28`
```

Comment:
1 example should be the complete command with end date (with time), planned date (with time), status and tags